plugin/completion.py

```diff
@@ -82,32 +82,36 @@ def update_popup(self, minihtml_content: str) -> None:
 
 class LspCompleteCommand(sublime_plugin.TextCommand):
 
-    def epilogue(self, item: Dict[str, Any]) -> None:
+    def epilogue(self, item: Dict[str, Any], session_name: Optional[str] = None) -> None:
         additional_edits = item.get('additionalTextEdits')
         if additional_edits:
             edits = [parse_text_edit(additional_edit) for additional_edit in additional_edits]
             self.view.run_command("lsp_apply_document_edit", {'changes': edits})
         command = item.get("command")
         if command:
             debug('Running server command "{}" for view {}'.format(command, self.view.id()))
-            args = {"command_name": command["command"], "command_args": command.get("arguments")}
+            args = {
+                "command_name": command["command"],
+                "command_args": command.get("arguments"),
+                "session_name": session_name
+            }
             self.view.run_command("lsp_execute", args)
 
 
 class LspCompleteInsertTextCommand(LspCompleteCommand):
 
-    def run(self, edit: sublime.Edit, **item: Any) -> None:
+    def run(self, edit: sublime.Edit, item: Any, session_name: Optional[str] = None) -> None:
```

Comment:
Not sure why there were `kwargs` used here. That would "collect" all unrelated arguments into the same argument, right? Don't understand the use case...

Reply:
It was for performance because it doesn't allocate another dict. But then again, ST serializes everything to a string and stores those arguments somewhere in plugin_host, so it probably doesn't matter much.